libraries/AP_Scripting/examples/standby-sim.lua

```diff
@@ -0,0 +1,80 @@
+-- This script is a allow to switch the standby function by the switch of the remote controller on 2 simulated fcus.
+
+-- variable to limit the number of messages sent to GCS
+local last_value = 0
+local last_rc_input = 0
+
+-- constants
+local switch_high = 2
+local switch_low = 0
+local standby_function = 76
+local rc_input_pin = 8
+
+
+-- for fast param access it is better to get a param object,
+-- this saves the code searching for the param by name every time
+local JSON_MASTER = Parameter()
+if not JSON_MASTER:init('SIM_JSON_MASTER') then
+  gcs:send_text(6, 'get JSON_MASTER failed')

```

Comment:
Looking pretty good.  I'd probably use the entire variable name, "SIM_JSON_MASTER" instead of shortening it because it will help users more quickly identify the issue